code/pizzabot.js

```diff
@@ -5,3 +5,38 @@ const pepperoni = "Pepperoni Pizza"
 const pizzaPrice = 80
 
 //Put your Javscript code here:
+
+let checkOrderName = (orderName) => {
+  if (
+    orderName === vegetarian || orderName === hawaiian || orderName === pepperoni
+  ) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+let orderTotal = (orderQuantity) => {
+  return +orderQuantity * pizzaPrice
```

Comment:
To make it from a string (the input from the prompt) to a number. It showed undefined at first. But maybe it is already done earlier in the code so it would not be needed this time?

Reply:
It's needed if you have `orderQuantity` as a string before (this converts it to a integer), so that's is probably the reason! Then it's good to have this